Examples/Scripts/Python/truth_tracking_chi2.py

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env python3
+import os
+from pathlib import Path
+from typing import Optional, Union
+
+import acts
+import acts.examples
+
+u = acts.UnitConstants
+
+
+def addChi2Tracks(
```

Comment:
can this be moved to `reconstruction.py`?

Reply:
should be moved there in the future, but for now I would like to keep it strictly separated.